docs/commands.md

```diff
@@ -68,14 +68,14 @@ TS.ALTER temperature:2:32 LABELS sensor_id 2 area_id 32 sub_area_id 15
 
 ### TS.ADD
 
-Append (or create and append) a new sample to the series.
+Append, or create and append, a new sample to the series that has been created earlier with TS.CREATE. 
```

Comment:
This is not accurate, you can add to a non existing serie

Reply:
Ok thank you for the clarification. What is the difference if the series is created with TS.CREATE before vs. you need to the non-existing series?

Reply:
@miohtama there's no big difference, you can control the configuration of the automatically created keys with MODULE ARGS or have full control using TS.CREATE

Reply:
Thank you @danni-m. I am not 100% following you. Are you referring to MODULE LOAD arguments or to something else?